client/my-sites/people/people-list-item/index.jsx

```diff
@@ -13,22 +12,23 @@ import CompactCard from 'components/card/compact';
 import PeopleProfile from 'my-sites/people/people-profile';
 import analytics from 'lib/analytics';
 import config from 'config';
+import { getSelectedSite } from 'state/ui/selectors';
+import { localize } from 'i18n-calypso';
 
-export default React.createClass( {
-
-	displayName: 'PeopleListItem',
-
-	mixins: [ PureRenderMixin ],
+class PeopleListItem extends React.PureComponent {
+	constructor( props ) {
+		super( props );
+	}
```

Comment:
We can use the default constructor and remove this if we aren't binding functions or setting initial state.